core/domain/stats_services.py

```diff
@@ -406,10 +406,10 @@ def update_exp_issues_for_new_exp_version(
                 playthrough.issue_customization_args['state_name']['value'] = (
                     new_state_name)
             for a_idx, action in enumerate(playthrough.actions):
-                if action.action_customization_args['state_name']['value'] == (
+                if action.actionCustomizationArgs['stateName']['value'] == (
```

Comment:
Please do not change the names of python variables, only the keys we pass to the fronntend ('stateName', in this example).

Reply:
Ok, I will rename `actionCustomizationArgs` to `action_customization_args` and all instances like this in this file and else where I see like this.